tests/test_apps.py

```diff
@@ -135,6 +135,23 @@ def setUp(self):
     def test_name(self):
         assert self.app_config.name == 'canvas'
 
+@mark.django_db
+class TestBlackboardConfig(unittest.TestCase):
+    """
+    Test integrated_channels.blackboard app config.
+    """
+
+    def setUp(self):
+        """
+        Set up test environment
+        """
+        super(TestBlackboardConfig, self).setUp()
+        self.app_config = integrated_channels.blackboard.apps.BlackboardConfig(
+            'blackboard', integrated_channels.blackboard
+        )
+
+    def test_name(self):
+        assert self.app_config.name == 'blackboard'
```

Comment:
Again, I'm tempted to avoid "magic strings" and have "blackboard" stored somewhere, but again, not critical.

Reply:
yup I like the idea, perhaps when I next work on this blackboard area, I will take care of that ... all channels could use that improvement